Projects/views.py

```diff
@@ -354,3 +334,32 @@ def change_label(request):
         issue.save()
         return HttpResponse(res)
     else: return HttpResponseBadRequest()
+
+def new_issue(request):
+    print('adding issue')
+    if request.method == 'POST':
+        if request.user.is_staff:
+            print('in add issue method')
```

Comment:
see this 
```
            new_issue = Issues()
            new_issue.mentor = mentor
            new_issue.title_issue = title_issue
            new_issue.link_issue = link_issue
            new_issue.title_project = title_project
            new_issue.link_project = link_project
            new_issue.level = level
            new_issue.points = points
            new_issue.save()
```

Reply:
Okay sir I will remeber this. This is a good practice. :+1: